backend/handlers/institution_children_handler.py

```diff
@@ -52,52 +52,49 @@ class InstitutionChildrenHandler(BaseHandler):
     @json_response
     @login_required
     @ndb.transactional(xg=True)
-    def delete(self, user, institution_key, institution_link):
+    def delete(self, user, institution_children_urlsafe, institution_parent_urlsafe):
         """
-        Handle delete children link between institutions.
+        Handle delete parent link between institutions.
 
-        This handler remove the children link between two institutions. 
-        If the parameter isParent is true, it means that the removal 
-        request has been made from a child institution, otherwise 
-        the request has been made by a parent institution.
+        This handler remove the parent link between two institutions.
```

Comment:
explain who makes the action, institution children send to remove parent or the opposite